### PR TITLE
created first entity - Book

### PR DESCRIPTION
**Pull request by Hryniv**
created:
- Book class
- Book's dao and service layers 
- DataProcessingException for dao layer
- tests
- aplication.properties files for tests and main program

### CODE REVIEW

File: src/main/java/com/bookstore/BookStoreApplication.java [hunk 1]
```diff
@@ -1,13 +1,34 @@
...
+    private BookService bookService;
 
     public static void main(String[] args) {
         SpringApplication.run(BookStoreApplication.class, args);
     }
 
+    @Bean
+    public CommandLineRunner commandLineRunner() {
+        return args -> {
+            //Book book = new Book();
```
Comment (Elena-Bruyako) [inline src/main/java/com/bookstore/BookStoreApplication.java]: Uncomment code

File: src/main/java/com/bookstore/model/Book.java [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+import jakarta.persistence.Column;
+import jakarta.persistence.Entity;
+import jakarta.persistence.GeneratedValue;
+import jakarta.persistence.GenerationType;
+import jakarta.persistence.Id;
+import java.math.BigDecimal;
+import lombok.Data;
+
+@Entity
+@Data
```
Comment (Elena-Bruyako) [inline src/main/java/com/bookstore/model/Book.java]: Don't use `@Data` for entities
Add `@Table` with name

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
Good job! Let's improve your solution

File: src/main/java/com/bookstore/model/Book.java [hunk 3]
```diff
@@ -0,0 +1,123 @@
...
+    @Column(nullable = false)
+    private String author;
+    @Column(nullable = false, unique = true)
+    private String isbn;
+    @Column(nullable = false)
+    private BigDecimal price;
+    private String description;
+    private String coverImage;
+
+    public Book() {
```
Comment (Elena-Bruyako) [inline src/main/java/com/bookstore/model/Book.java]: Read which annotations `@Data` includes and use required

**Review by Elena-Bruyako — CHANGES_REQUESTED** [review]
see comment

**Review by okuzan — APPROVED** [review]

**Pull request merged**